index.d.ts

```diff
@@ -269,7 +269,7 @@ declare namespace RNRF {
     jump(props: props): void,
     refresh(props: props): void,
     focus(props: props): void,
-    create(scene: Scene, wrapBy?: () => any): Object
+    create(scene: React.ReactNode, wrapBy?: () => any): Object
```

Comment:
`scene `here is extends from `React.Component` it should works .By the way. are you using `typescript` with `react-native`.If so can you provide an online version,I can have a look.